smoker.spec

```diff
@@ -0,0 +1,58 @@
+%global branch master
+%global candidate rc2
```

Comment:
I'm not aware of a way how to generate the whole specfile. The rpm dependencies have different names than the python eggs, so I can not image where an automated specfile generation tool would get these from. 
I guess we can rewrite these fields from the makefile (not sure if the specfile is not being read by koji before this step), or vice versa, propagate the version from specfile to setup.py.